### ISSUE TITLE
getStatus: e.isUser is not a function

### ISSUE THREAD

**Issue by enodr** [orkestral/venom#1687]
## Description

Since a couple of days most (all?) operations like getStatus, getNumberProfile, etc are failing with errors like e.isUser is not a function

## Environment

- **Venom version(s):** 4.0.11
- **Browser:** google-chrome-stable_98.0.4758.80-1_amd64.deb
- **OS:** Ubuntu 20.04
- **Node version:** Node 17.4.0

## Steps to Reproduce

1. Create session
2. Call client.getStatus(id);

## Log Output

```
whatsapp_1    | error caught in http server:  Error: Evaluation failed: TypeError: e.isUser is not a function
whatsapp_1    |     at https://web.whatsapp.com/bootstrap_qr.a93ca7c63be0f2cdef56.js:32:230318
whatsapp_1    |     at Generator.next (<anonymous>)
whatsapp_1    |     at t (https://web.whatsapp.com/vendor1~bootstrap_qr.27ae36d14a5e15ff6df9.js:2:65356)
whatsapp_1    |     at s (https://web.whatsapp.com/vendor1~bootstrap_qr.27ae36d14a5e15ff6df9.js:2:65567)
whatsapp_1    |     at https://web.whatsapp.com/vendor1~bootstrap_qr.27ae36d14a5e15ff6df9.js:2:65626
whatsapp_1    |     at Y (https://web.whatsapp.com/bootstrap_qr.a93ca7c63be0f2cdef56.js:31:99298)
whatsapp_1    |     at new y (https://web.whatsapp.com/bootstrap_qr.a93ca7c63be0f2cdef56.js:31:91865)
whatsapp_1    |     at https://web.whatsapp.com/vendor1~bootstrap_qr.27ae36d14a5e15ff6df9.js:2:65507
whatsapp_1    |     at l (https://web.whatsapp.com/bootstrap_qr.a93ca7c63be0f2cdef56.js:32:230871)
whatsapp_1    |     at https://web.whatsapp.com/bootstrap_qr.a93ca7c63be0f2cdef56.js:32:230164
whatsapp_1    |     at ExecutionContext._evaluateInternal (/app/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
whatsapp_1    |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
whatsapp_1    |     at async ExecutionContext.evaluate (/app/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)
whatsapp_1    |     at async get_info (/app/index.js:156:21)
whatsapp_1    |     at async Server.<anonymous> (/app/index.js:117:18)
```

**Comment by orkestral** [comment]
Thank you for joining our community, happy. We are currently fixing this bug, our new version will be released soon.

**Comment by orkestral** [comment]
**Fix in version 4.1.0**
